pages/profile.tsx

```diff
@@ -68,73 +68,77 @@ export default function ProfilePage() {
-                  <div className="font-bold text-xl">Name</div>
-                  <h1 className="font-bold">{`${profile.user.firstName} ${profile.user.lastName}`}</h1>
-                </div>
-                <div className="profile-view-role flex flex-col gap-y-2">
-                  <div className="font-bold text-xl">Role</div>
-                  <h1 className="font-bold">{profile.user.permissions[0]}</h1>
-                </div>
-                <div className="profile-view-univ flex flex-col gap-y-2">
-                  <div className="font-bold text-xl">University</div>
-                  <h1 className="font-bold">{profile.university}</h1>
-                </div>
-                <div className="profile-view-major flex flex-col gap-y-2">
-                  <div className="font-bold text-xl">Major</div>
-                  <h1 className="font-bold">{profile.major}</h1>
-                </div>
-                <div className="profile-view-stlvl flex flex-col gap-y-2">
-                  <div className="font-bold text-xl">Level of Study</div>
-                  <h1 className="font-bold">{profile.studyLevel}</h1>
-                </div>
-                <div>
-                  {!uploading ? (
-                    <>
-                      <input
-                        id="resume"
-                        style={{ display: 'none' }}
-                        type="file"
-                        ref={resumeRef}
-                        onChange={() => handleResumeUpload(profile)}
-                        accept=".pdf, .doc, .docx, image/png, image/jpeg, .txt, .tex, .rtf"
-                      />
-                      <label
-                        id="resume_label"
-                        className="transition rounded p-3 text-center whitespace-nowrap text-white w-min bg-gray-500 cursor-pointer font-black gap-y-2 hover:brightness-110"
-                        htmlFor="resume"
-                      >
-                        Update Resume
-                      </label>
-                    </>
-                  ) : (
-                    <LoadIcon width={16} height={16} />
-                  )}
-                </div>
-              </div>
-            </div>
+    <div className="xl:px-36 lg:px-28 sm:px-16 px-10 md:py-16 py-12 ">
+      <div className="flex flex-col md:flex-row 2xl:gap-x-48 gap-x-20 2xl:justify-center">
+        {/* QR Code */}
+        <div className="">
+          <div className="bg-secondary rounded-lg p-8 h-min w-min mx-auto">
+            {/* Dark represents dots, Light represents the background */}
+            <QRCode
+              data={'hack:' + user.id}
+              loading={false}
+              width={200}
+              height={200}
+              darkColor="#05149C"
+              lightColor="#0000"
+            />
           </div>
-        </section>
+          <div className="text-complementary border-y-[1px] border-primaryDark py-4 md:my-8 my-6">
```

Comment:
color and size doesnt match figma it should be `border-primaryDark/20` and `border-y-[1.2px]`